docs/enroute/KAI.md

```diff
@@ -90,6 +90,15 @@ KAI may clear aircraft direct to the STAR's CH TMA boundary fix without coordina
 
     **Note**: As `KABGO` sits within CH TMA's airspace, coordination would be required.
 
+#### Short STARs - NZCH
```

Comment:
I don't think mentioning of the `# K` and `# J` STARs is necessary within the KAI page, as the STAR's commencement is within the STH sector. Thoughts?

Reply:
Will move to STH